src/renderer/component/fileTile/view.jsx

```diff
@@ -108,43 +134,38 @@ class FileTile extends React.PureComponent<Props> {
-          {!isResolvingUri && (
+          <div className="file-tile__title">
+            {(title || name) && <TruncatedText text={title || name} lines={size === 'small' ? 2 : 3} />}
+          </div>
+          <div className="card__subtitle">
+            <UriIndicator uri={uri} link />
+          </div>
+          <div className="card__subtitle card--space-between">
+            <DateTime timeAgo block={height} />
+            {size !== 'large' && this.renderFileProperties()}
+          </div>
+          {displayDescription && (
+            <div className="card__subtext">
+              <TruncatedText text={description} lines={size === 'large' ? 4 : 3} />
+            </div>
+          )}
+          {size === 'large' && this.renderFileProperties()}
+          {!name && (
             <React.Fragment>
-              <div className="file-tile__title">
-                <TruncatedText text={title || name} lines={size === 'small' ? 2 : 3} />
-              </div>
-              <div className="card__subtitle">
-                {showUri ? uri : <UriIndicator uri={uri} link />}
-              </div>
-              <div className="card__file-properties">
-                <FilePrice hideFree uri={uri} />
-                {isRewardContent && <Icon iconColor="red" icon={icons.FEATURED} />}
-                {showLocal && isDownloaded && <Icon icon={icons.LOCAL} />}
-              </div>
-              {displayDescription && (
-                <div className="card__subtext">
-                  <TruncatedText text={description} lines={size === 'large' ? 4 : 3} />
-                </div>
-              )}
-              {!name && (
-                <React.Fragment>
-                  {__('This location is unused.')}{' '}
-                  <Button
-                    button="link"
-                    label={__('Put something here!')}
-                    onClick={e => {
-                      // avoid navigating to /show from clicking on the section
-                      e.stopPropagation();
-                      // strip prefix from the Uri and use that as navigation parameter
-                      const nameFromUri = uri.replace(/lbry:\/\//g, '').replace(/-/g, ' ');
-                      clearPublish(); // to remove any existing publish data
-                      updatePublishForm({ name: nameFromUri }); // to populate the name
-                      navigate('/publish');
-                    }}
-                  />
-                </React.Fragment>
-              )}
+              {__('This location is unused.')}{' '}
+              <Button
+                button="link"
+                label={__('Put something here!')}
+                onClick={e => {
+                  // avoid navigating to /show from clicking on the section
+                  e.stopPropagation();
+                  // strip prefix from the Uri and use that as navigation parameter
+                  const nameFromUri = uri.replace(/lbry:\/\//g, '').replace(/-/g, ' ');
```

Comment:
Why is this logic here?  Seems it should be shared logic

Reply:
👍 We should probably use `const { name } = parseUri(uri)`